pelican/contents.py

```diff
@@ -380,6 +377,23 @@ def summary(self, value):
         """Dummy function"""
         pass
 
+    @property
+    def _summary(self):
+        """another deprecated function to access summary"""
+
+        logger.warning('_summary has been deprecated since 3.8.0. '
+                       'Use the summary decorator instead')
+        return self.summary
+
+    @_summary.setter
+    def _summary(self, value):
+        """another deprecated function to access summary"""
+
+        logger.warning('_summary has been deprecated since 3.8.0. '
+                       'Use the summary decorator instead')
```

Comment:
summary *decorator*? You mean property ('ie. `.summary`)? or `metadata['summary']`?

First is a no go. The reason for someone using `._summary` is (or at least should be) because they can't use `.summary` for various reasons like intrasite links etc. 

And this is getting out of hand :). `._summary` was an internal variable for `.summary` property. Now, it's becoming property.

Also, I don't get why you need this. You don't need `refresh_metadata_intersite_links()` working on `summary` because it already has that handled via the `.summary` property. If you want to treat `summary` as any other metadata and rely on `refresh_metadata_intersite_links()` then the `.summary` getter need to be changed to not do that as well. Or if you want to keep things as is, `summary` can be excluded from `refresh_metadata_intersite_links()`.

Either way, there is some redundancy here.

Reply:
> And this is getting out of hand :)

yes, sorry :see_no_evil: 

Hm, okay, so the only other option I see is to put back the `_summary`  internal variable and related parts in `__init__()` / `get_summary()` and in addition also explicitly process it in `refresh_metadata_intersite_links()`:

```diff
diff --git a/pelican/contents.py b/pelican/contents.py
index 09d388d4..3cae9acf 100644
--- a/pelican/contents.py
+++ b/pelican/contents.py
@@ -447,7 +447,7 @@ class Content(object):
 
     def refresh_metadata_intersite_links(self):
         for key in self.settings['FORMATTED_FIELDS']:
-            if key in self.metadata:
+            if key in self.metadata and key != 'summary':
                 value = self._update_content(
                     self.metadata[key],
                     self.get_siteurl()
@@ -455,6 +455,14 @@ class Content(object):
                 self.metadata[key] = value
                 setattr(self, key.lower(), value)
 
+        # _summary is an internal variable that some plugins may be writing to,
+        # so ensure changes to it are picked up
+        if 'summary' in self.settings['FORMATTED_FIELDS'] and 'summary' in self.metadata:
+            self._summary = self._update_content(
+                self._summary,
+                self.get_siteurl()
+            )
+            self.metadata['summary'] = value
 
 class Page(Content):
     mandatory_properties = ('title',)

```

Does this look better to you?